extensions/amp-base-carousel/amp-base-carousel.md

```diff
@@ -128,6 +128,14 @@ useful when using `visible-count`. This
 
 #### Miscellaneous
 
+##### controls
+
+Either `"always"`, `"auto"`, or `"never"`, defaults to `"auto"`. This determines if and when prev/next navigational arrows are displayed. Note: When `outset-arrows` is `true`, the arrows are shown `"always"`.
+
+-   `always`: Arrows are always displayed.
+-   `auto`: Arrows are displayed when a mouse has interacted with the carousel, and
```

Comment:
nit: carousel.

Reply:
Good catch! Updating this in the other md file I copied this from as well. 😂

Reply:
Actually there was more to this sentence so added the rest now.